apps/ledger-live-mobile/package.json

```diff
@@ -12,6 +12,7 @@
     "ios": "react-native run-ios",
     "android": "react-native run-android --appIdSuffix=debug",
     "android:hermes": "export HERMES_ENABLED_ANDROID=true && yarn android",
+    "detox": "./node_modules/.bin/detox",
```

Comment:
Is this required or just a leftover from debugging?

Reply:
It is required for the pnpm detox commands to work.
Ex : pnpm detox clean-framework-cache